tests/plugins/emoji/manual/repeated.html

```diff
@@ -0,0 +1,10 @@
+<h2>Classic editor</h2>
+<div id="classic"></div>
+<h2>Inline editor</h2>
+<div id="inline" contenteditable="true">
```

Comment:
Just a detail, but since you're closing all of the tags in a single line, it would be nice to close this one as well.